docs/deployments.md

```diff
@@ -73,31 +81,34 @@ N/A
+  - Manually build the Docker image and push it to a registry that is accessible from your cluster (Docker Hub, IBM Cloud Container Registry, manually deployed Quay instance):
     - `docker build -t <private-registry>/<image-namespace>/order-command-ms:latest order-command-ms/`
     - `docker build -t <private-registry>/<image-namespace>/order-query-ms:latest order-query-ms/`
     - `docker login <private-registry>`
     - `docker push <private-registry>/<image-namespace>/order-command-ms:latest`
     - `docker push <private-registry>/<image-namespace>/order-query-ms:latest`
-3. Generate application YAMLs via `helm template`:
+2. Generate application YAMLs via `helm template` for both `order-command` and `order-query`:
   - Parameters:
     - `--set image.repository=<private-registry>/<image-namespace>/<image-repository>`
-    - `--set image.tag=latest`
     - `--set image.pullSecret=<private-registry-pullsecret>` (optional or set to blank)
-    - `--set image.pullPolicy=Always`
-    - `--set eventstreams.env=ICP`
-    - `--set eventstreams.brokersConfigMap=<kafka brokers ConfigMap name>`
+    - `--set kafka.brokersConfigMap=<kafka brokers ConfigMap name>`
+    - `--set eventstreams.enabled=(true/false)` (`true` when connecting to Event Streams of any kind, `false` when connecting to Kafka directly)
+    - `--set eventstreams.apikeyConfigMap=<kafka api key Secret name>`
+    - `--set eventstreams.truststoreRequired=(true/false)` (`true` when connecting to Event Streams via ICP4I)
+    - `--set eventstreams.truststoreSecret=<eventstreams jks file secret name>` (only used when connecting to Event Streams via ICP4I)
+    - `--set eventstreams.truststorePassword=<eventstreams jks password>` (only used when connecting to Event Streams via ICP4I)
     - `--set serviceAccountName=<service-account-name>`
     - `--namespace <target-namespace>`
     - `--output-dir <local-template-directory>`
-  - Example: `helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-command-ms --set image.tag=latest --set image.pullSecret= --set image.pullPolicy=Always --set eventstreams.env=ICP --set eventstreams.brokersConfigMap=kafka-brokers --set serviceAccountName=kcontainer-runtime --output-dir templ --namespace eda-refarch chart/ordercommandms/`
+  - Example using Event Streams via ICP4I:
+   ```shell
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-command-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-pipelines-sandbox chart/ordercommandms
```

Comment:
shouldn't `eventstreams.truststoreSecret=es-ca-pemfile` be  `eventstreams.truststoreSecret=es-truststore-jks` ??

Reply:
You are correct.  Fixed this in the commit below -> https://github.com/ibm-cloud-architecture/refarch-kc-order-ms/pull/25/commits/04cd9b07d8c3c2b9e56ba2c4ddbc377377386beb